dashboards/go/go-runtime-view-prometheus.json

```diff
@@ -0,0 +1,626 @@
+              "showLegend": false,
+              "displayHorizontal": false
+            }
+          },
+          "id": ""
+        }
+      },
+      {
+        "yPos": 68,
+        "width": 24,
+        "height": 13,
+        "widget": {
+          "title": "GC Stop-The-World 100th percentile latency",
+          "xyChart": {
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "prometheusQuery": "sum by(project_id, location, cluster, job, namespace, instance)(go_gc_duration_seconds{quantile=~\"1|1.0\", job=~\"${job.value}\", namespace=~\"${namespace.value}\", instance=~\"${instance.value}\"})",
+                  "unitOverride": "s",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "",
+                "targetAxis": "Y1",
+                "dimensions": [],
+                "measures": [],
+                "breakdowns": []
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            },
+            "chartOptions": {
+              "mode": "COLOR",
+              "showLegend": false,
+              "displayHorizontal": false
+            }
+          },
+          "id": ""
+        }
+      },
+      {
+        "xPos": 24,
+        "yPos": 68,
+        "width": 24,
+        "height": 13,
+        "widget": {
+          "title": "GC Heap Bytes Next Goal",
+          "xyChart": {
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "prometheusQuery": "sum by(project_id, location, cluster, job, namespace, instance)(go_memstats_next_gc_bytes{job=~\"${job.value}\", namespace=~\"${namespace.value}\", instance=~\"${instance.value}\"})",
+                  "unitOverride": "By",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "",
+                "targetAxis": "Y1",
+                "dimensions": [],
+                "measures": [],
+                "breakdowns": []
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            },
+            "chartOptions": {
+              "mode": "COLOR",
+              "showLegend": false,
+              "displayHorizontal": false
+            }
+          },
+          "id": ""
+        }
+      },
+      {
+        "yPos": 30,
+        "width": 48,
+        "height": 51,
+        "widget": {
+          "title": "Memory",
+          "collapsibleGroup": {
+            "collapsed": false
+          },
+          "id": ""
+        }
+      },
+      {
+        "yPos": 10,
+        "width": 48,
+        "height": 8,
+        "widget": {
+          "title": "Runtime Configuration",
+          "collapsibleGroup": {
+            "collapsed": true
```

Comment:
Just double-checking if it's intentional which sections you have collapsed or uncollapsed by default on page load

Reply:
Correct, those are useful only in certain, less often cases (but still important enough to have those on dashboard as per https://github.com/golang/go/issues/67120)